src/app/components/plugin-manager-component.js

```diff
@@ -164,36 +179,36 @@ class PluginManagerComponent extends ViewPlugin {
 
     const activeTile = actives.length !== 0
       ? yo`
-      <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between align-items-center">
-        <span class="navbar-brand">Active Modules</span>
-        <span class="badge badge-pill badge-primary">${actives.length}</span>
+      <nav class="plugins-list-header justify-content-start navbar navbar-expand-lg bg-light navbar-light align-items-center px-4 py-3">
+        <span class="h6 mb-0 mr-2">Active Modules</span>
+        <span class="badge badge-pill badge-warning">${actives.length}</span>
       </nav>`
       : ''
     const inactiveTile = inactives.length !== 0
       ? yo`
-      <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between align-items-center">
-        <span class="navbar-brand">Inactive Modules</span>
-        <span class="badge badge-pill badge-primary" style = "cursor: default;">${inactives.length}</span>
+      <nav class="plugins-list-header justify-content-start navbar navbar-expand-lg bg-light navbar-light align-items-center px-4 py-3">
```

Comment:
px-4 you added here is braking other themes, can't you fix this in nav classes definitions in a theme directly?